docs/maintain/hosting.rst

```diff
@@ -4,6 +4,80 @@ Hosting
 Rescale a server
 ----------------
 
-The Bytemark Panel makes it easy to scale/resize a server (number of cores and GiB of RAM).
+The Bytemark and Linode Control Panels makes it easy to scale/resize a server (number of cores and GiB of RAM).
 
 You must :doc:`deploy the service<../deploy/deploy>` to re-configure swap, Elasticsearch, PostgreSQL and/or uWSGI.
+
+Recovery
+--------
+
+In the event that a server is inaccessible including via SSH there are a number of recovery systems we can use to restore access.
+
+The first step regardless of the ISP is to initiate a reboot via the server provider.
+This will clear down the current server state (unsaved firewall rules, running processes and system resources (CPU / Memory)), often this will bring the server back online.
+
+Linode
+^^^^^^
+LISH (Linode Shell) is a console hosted by Linode providing direct access to our servers.
+Using the LISH console we can login directly as if we were connecting to the instance locally.
+
+To access LISH:
+#. Log into Linode
+#. Select the server you want to access
+#. Select *Launch LISH Console*
+#. Connect as the ``root`` user using the password stored in `LastPass <https://www.lastpass.com>`__.
+
+`Linode documentation including SSH tunnelling with LISH <https://www.linode.com/docs/guides/using-the-lish-console/>`__.
+
+Hetzner
+^^^^^^^
+The Hetzner rescue system works by booting the server onto a temporary recovery image, from here we can mount the server disks and fix issues directly.
+
+To set up the rescue system:
+#. `Log into Hetzner <https://robot.your-server.de/server>`__
+#. Select the server you want to access
+#. Enable the Rescue System, select *Rescue*
+   #. Operating system: Linux
+   #. Architecture: 64 bit
+   #. Public key: Select your key
+   #. *Activate rescue system*
+
+   The rescue system will now be used next time we reboot.
+
+.. Note::
+
+   If your key is not in Hetzner you can upload it under the `*Key management* page<https://robot.your-server.de/key/index>`__.
+
+#. Reboot the server
+   #. Select the *Reset* heading
+   #. *Send CTRL+ALT+DEL to the server*
```

Comment:
I don't see this option.

<img width="613" alt="Screen Shot 2022-05-10 at 1 12 03 PM" src="https://user-images.githubusercontent.com/26463/167684782-fcbc73f3-484e-43b7-a064-a1681d8c2697.png">

Reply:
It seems this setting depends on the server (or perhaps which datacentre it is in).
I've updated the docs to list both options.

![Screenshot 2022-05-18 at 11-20-13 Hetzner Online GmbH - Robot](https://user-images.githubusercontent.com/8707878/169018030-c4179865-3920-4f59-b3a8-71a6b4122ca0.png)